packages/grafana-ui/src/components/Menu/Menu.tsx

```diff
@@ -36,7 +39,7 @@ export interface MenuProps extends React.HTMLAttributes<HTMLDivElement> {
   onClose?: () => void;
 }
 
-/** @public */
+/** @alpha */
```

Comment:
Moving from public to alpha seems a bit strange as this component might be already used publicly?

Reply:
this was recently added in 7.4 and only used by ContextMenu, I do not like it's API. WIll try to get this change to alpha  into 7.4